frontend/src/components/global/AlgoDraggable.vue

```diff
@@ -1,20 +1,21 @@
 <template>
-    <div :id="id" class="algo-draggable-container">
+    <div :id="id" class="algo-draggable-container" :class="additionalClass">
         <TransitionGroup name="algo-draggable-group">
             <v-chip
                 closable
-                v-for="variable in this.selectedVariables"
```

Comment:
`variable` to jest zaszłość z tego, że wcześniej ta lista była używana wcześniej tylko dla zmiennych. Proponuję pozmieniać nazewnictwo w tym pliku na bardziej neutralne. Proponuję nomenklaturę `element` zamiast `variable`